extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/runtime/MongoClientSupport.java

```diff
@@ -1,21 +1,35 @@
 package io.quarkus.mongodb.runtime;
 
+import java.util.ArrayList;
 import java.util.List;
 
+import com.mongodb.event.CommandListener;
 import com.mongodb.event.ConnectionPoolListener;
 
 public class MongoClientSupport {
 
     private final List<String> codecProviders;
     private final List<String> bsonDiscriminators;
     private final List<ConnectionPoolListener> connectionPoolListeners;
+    private final List<CommandListener> commandListeners;
     private final boolean disableSslSupport;
 
     public MongoClientSupport(List<String> codecProviders, List<String> bsonDiscriminators,
             List<ConnectionPoolListener> connectionPoolListeners, boolean disableSslSupport) {
         this.codecProviders = codecProviders;
         this.bsonDiscriminators = bsonDiscriminators;
         this.connectionPoolListeners = connectionPoolListeners;
+        this.commandListeners = new ArrayList<>();
+        this.disableSslSupport = disableSslSupport;
+    }
+
+    public MongoClientSupport(List<String> codecProviders, List<String> bsonDiscriminators,
+            List<ConnectionPoolListener> connectionPoolListeners, List<CommandListener> commandListeners,
+            boolean disableSslSupport) {
+        this.codecProviders = codecProviders;
+        this.bsonDiscriminators = bsonDiscriminators;
+        this.connectionPoolListeners = connectionPoolListeners;
+        this.commandListeners = commandListeners;
```

Comment:
I don't think this is needed to provides methods with and without the List of CommandListener. This method is only used inside the deployment module so it's signature can be modified.